.travis.yml

```diff
@@ -24,6 +24,7 @@ script:
   # some tests are randomized. Run them a few times.
   - for i in `seq 1 10`; do
       ginkgo -r -v --cover --randomizeAllSpecs --randomizeSuites --trace --progress;
+      ginkgo -tags openssl -r -v --cover --randomizeAllSpecs --randomizeSuites --trace --progress;
```

Comment:
This should use the [Travis build matrix](https://docs.travis-ci.com/user/build-matrix/) feature.